sections/header.liquid

```diff
@@ -71,10 +71,18 @@
     margin-bottom: {{ section.settings.margin_bottom | times: 0.75 | round: 0 }}px;
   }
 
```

Comment:
That's interesting - it looks like what's happening is that you're ending up with the sticky header receding.  It looks like you're scrolling because your screen is smaller than the total size of the header + the notification.

If you test with a larger screen that shouldn't happen - I think a screen that small is an edge case.

Reply:
Yes! It was only when I had my console open